### ISSUE TITLE
How to fit a model with two types of tagging(output)?

### ISSUE THREAD

**Issue by trisha-git** [xuxingya/tf2crf#43]
Hello,

I am trying to implement a NER where for an input I would like to receive two outputs. I have multiple questions for the same.

1. This is arising some assertion error in ModelWithCRFLoss. My current code snippet 
```
    inp = Input(shape=(max_len,)) 
    model = Embedding(input_dim=n_chars + 1, output_dim=16, input_length=max_len, mask_zero=True)(inp) 
    bi_lstm = Bidirectional(LSTM(units=16, return_sequences=True, recurrent_dropout=0.2))(model)  
    crf_0 = CRF(n_tags[0]+1)
    crf_1 = CRF(n_tags[1]+1)
    out_0 = crf_0(bi_lstm)  
    out_1 = crf_1(bi_lstm)  
    base_model = Model(inp, [out_0,out_1])
    #loss is crf loss, accuracy is crf accuracy
    model = ModelWithCRFLoss(base_model, sparse_target=True)
    model.compile(optimizer="rmsprop")
    history = model.fit(X_tr, [y_tr_0, y_tr_1], batch_size=my_batch_size, epochs=my_epochs, validation_split=0.1, verbose=1)
```
2. I also tried training the model using single output which works. But, while trying to save the model in a json format as described in a closed issue I am getting a NotImplementedError from get_config() as the ModelWithCRFLoss do not have a get_config().
3. Is it possible to not use the ModelWithCRFLoss and use crf loss directly from CRF?I tried to follow a [closed old issue](https://github.com/xuxingya/tf2crf/issues/8), but the CRF object do not have a loss function any more.

**Comment by xuxingya** [comment]
I am sorry that currently I am not able to test this problem. You can try implement a get_config() function according to this [tensorflow tutorial](https://www.tensorflow.org/guide/keras/custom_layers_and_models). It is simple.
